sass/components/_btn.scss

```diff
@@ -2,3 +2,10 @@
   line-height: 1;
   padding: .25rem 1rem;
 }
+
+.btn-primary:disabled,
+.btn-primary.is-disabled {
```

Comment:
@brendansudol what do you think about adding the `:hover` psuedo class to this style declaration? I imagine that a disabled button doesn't really respond to any of the user's mouse events so its sort of weird that it changes color on hover.